src/app/pages/MostReadPage/index.jsx

```diff
@@ -41,38 +36,20 @@ const ConstrainedWrapper = styled.div`
   }
 `;
 
-const MostReadHeader = styled.h1.attrs({
-  id: 'content',
-  tabIndex: '-1',
-})`
-  color: ${C_METAL};
-  ${({ script }) => script && getDoublePica(script)};
-  ${({ service }) => getSansRegular(service)};
-  margin: 0;
-  padding: ${GEL_SPACING_DBL} 0 ${GEL_SPACING_TRPL};
-  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
-    padding: ${GEL_SPACING_TRPL} 0 ${GEL_SPACING_DBL};
-  }
-  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
-    padding: ${GEL_SPACING_TRPL} 0 ${GEL_SPACING};
-  }
-`;
-
 const MostReadPage = ({ pageData, mostReadEndpointOverride }) => {
   const {
     brandName,
-    service,
-    script,
+
     dir,
     lang,
     mostRead: { header },
   } = useContext(ServiceContext);
 
   const MostReadWrapper = ({ children }) => (
     <ConstrainedWrapper>
-      <MostReadHeader script={script} service={service}>
+      <IndexHeading id="content" pageType="mostRead">
```

Comment:
Does this also need `tabIndex: '-1'` that was in Most Read's heading previously?

Reply:
This is already applied inside the component in Psammead https://github.com/bbc/psammead/blob/latest/packages/components/psammead-heading-index/src/index.jsx#L9